src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -2392,22 +2376,22 @@ def security_domain_download(cmd, client, hsm_name, sd_wrapping_keys, security_d
 
     certificates = []
     for path in sd_wrapping_keys:
-        sd_jwk = SecurityDomainJsonWebKey()
+        sd_jwk = {}
         with open(path, 'rb') as f:
             pem_data = f.read()
 
         cert = load_pem_x509_certificate(pem_data, backend=default_backend())
         public_key = cert.public_key()
         public_bytes = cert.public_bytes(Encoding.DER)
-        sd_jwk.x5c = [Utils.security_domain_b64_url_encode_for_x5c(public_bytes)]  # only one cert, not a chain
-        sd_jwk.x5t = Utils.security_domain_b64_url_encode(hashlib.sha1(public_bytes).digest())
-        sd_jwk.x5tS256 = Utils.security_domain_b64_url_encode(hashlib.sha256(public_bytes).digest())
-        sd_jwk.key_ops = ['verify', 'encrypt', 'wrapKey']
+        sd_jwk['x5c'] = [Utils.security_domain_b64_url_encode_for_x5c(public_bytes)]  # only one cert, not a chain
+        sd_jwk['x5t'] = Utils.security_domain_b64_url_encode(hashlib.sha1(public_bytes).digest())
+        sd_jwk['x5t#S256'] = Utils.security_domain_b64_url_encode(hashlib.sha256(public_bytes).digest())
+        sd_jwk['key_ops'] = ['verify', 'encrypt', 'wrapKey']
```

Comment:
Does Track 2 SDK now require `dict` instead of an SDK object?

Reply:
Track2 SDK can accept both model object and dict. I don't want to import many models from SDK so here use simple dict